spec/paperclip/attachment_registry_spec.rb

```diff
@@ -54,21 +54,60 @@
         greeter: { ciao: 'greeting' }
       }
       foo = Class.new
-      Paperclip::AttachmentRegistry.register(foo, :avatar, { yo: 'greeting' })
-      Paperclip::AttachmentRegistry.register(foo, :greeter, { ciao: 'greeting' })
+      Paperclip::AttachmentRegistry.register(
+        foo,
+        :avatar,
+        { yo: 'greeting' }
+      )
+      Paperclip::AttachmentRegistry.register(
+        foo,
+        :greeter,
+        { ciao: 'greeting' }
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.